extensions/authclient/BaseOAuth.php

```diff
@@ -397,7 +397,7 @@ protected function composeUrl($url, array $params = [])
     /**
      * Saves token as persistent state.
      * @param OAuthToken $token auth token
-     * @return static self reference.
+     * @return $this
```

Comment:
don't you think thats redundant? $this is a self reference by definition.

besides as you can see on this pr, there was no consistency about what comment to add.